doc/whats_new.rst

```diff
@@ -25,6 +25,9 @@ New features
    - Added the :class:`ensemble.IsolationForest` class for anomaly detection based on
      random forests. By `Nicolas Goix`_.
 
+   - Added new supervised learning algorithm: :ref:`Multi-layer Perceptron <multi-layer-perceptron>`
```

Comment:
I did try that target before but used dash instead of underscore, fixed it now. Thanks for the help :smiley: